Formula/wb32-dfu-updater_cli.rb

```diff
@@ -0,0 +1,31 @@
+class Wb32DfuUpdaterCli < Formula
+  desc "USB programmer"
+  homepage "https://github.com/WestberryTech/wb32-dfu-updater"
+  url "https://github.com/WestberryTech/wb32-dfu-updater/archive/refs/tags/v0.0.3.tar.gz"
+  sha256 "91865293c08368a8778789656b4a2c2e792c0a2b7238b42147d201e506c5de50"
+  license "Apache-2.0"
+  head "https://github.com/WestberryTech/wb32-dfu-updater.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "libusb"
+
+  def install
+    args = std_cmake_args
+
+    libusb = Formula["libusb"]
+    args << "-DLIBUSB_INCLUDE_DIRS=#{libusb.opt_include}/libusb-1.0"
+    args << "-DLIBUSB_LIBRARIES=#{libusb.opt_lib}/#{shared_library("libusb-1.0")}"
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make", "install"
+    end
+  end
+
+  test do
+    printing_message = "wb32-dfu-updater ver: #{version}\n \
+                        libusb version 1.0.24 (11584)\n \
+                        ----------------------------------------\n"
```

Comment:
```suggestion
    printing_message = <<~EOS
      wb32-dfu-updater ver: #{version}
      libusb version #{Formula["libusb"].version} (11584)
      ----------------------------------------
    EOS
```
Not quite sure where the `11584` comes from, but I don't think that's something we want to keep static here.